sdk/resources/Azure.Management.Resources/tests/Helpers/Utilities.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
License header should be added to each cs file

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
Done